### PR TITLE
FIX: Spaces after dots in translatable strings

### PR DESCRIPTION
**Pull request by Skif-off**
See diff.

### CODE REVIEW

File: workbench/src/dialogs.c [hunk 1]
```diff
@@ -443,7 +443,7 @@ gboolean dialogs_workbench_settings(WORKBENCH *workbench)
 	ui_table_add_row(GTK_TABLE(table), 1, w_enable_live_update, NULL);
 	gtk_widget_set_tooltip_text(w_enable_live_update,
 		_("If the option is activated (default), then the list of files and the sidebar"
-		  " will be updated automatically if a file or directory is created, removed or renamed."
+		  " will be updated automatically if a file or directory is created, removed or renamed. "
```
Comment (b4n) [inline workbench/src/dialogs.c]: Ideally the leading space here would also be on the previous line, but that's OK with this.

File: geanyvc/src/geanyvc.c [hunk 2]
```diff
@@ -1949,7 +1949,7 @@ plugin_configure(GtkDialog * dialog)
 	gtk_widget_set_tooltip_text(widgets.cb_editor_menu_entries,
 			     _("Whether menu for this plugin are getting placed either "
 			       "inside tools menu or directly inside Geany's menubar."
-			       "Will take in account after next start of GeanyVC"));
+			       " Will take in account after next start of GeanyVC"));
```
[the remark this answers is not in the document]
Comment (b4n) [reply geanyvc/src/geanyvc.c]: Forget my comment, it was absurd, I read the diff the wrong way around.

**Comment by Skif-off** [comment]
Done (fix + squash).

Sorry, I saw strings with space in beginning below/above and maked it so ([1](https://github.com/geany/geany-plugins/blob/master/workbench/src/dialogs.c#L455), [2](https://github.com/geany/geany-plugins/blob/master/workbench/src/dialogs.c#L455), [3](https://github.com/geany/geany-plugins/blob/master/workbench/src/dialogs.c#L455)).

File: workbench/src/dialogs.c (hunk 1 above)
Comment (Skif-off) [reply workbench/src/dialogs.c]: Will do it?

Comment (Skif-off) [reply workbench/src/dialogs.c]: Ops, merged.

Comment (frlan) [reply workbench/src/dialogs.c]: Did see the leading space to late. Will change it directly

**Comment by b4n** [comment]
I also updated the strings in the translations to include the space, as it doesn't have to be reviewed by translators (I made sure that the translations didn't have the same dot issue, and they don't): 65475dac74800f8e07d0102beeb64ed1a2612212

**Comment by LarsGit223** [comment]
I added a small section "Translation" to geany's HACKING file to prevent others from making the same fault as me.

**Pull request merged**